Makefile

```diff
@@ -95,15 +95,15 @@ _go_webdriver_test: var-BROWSER java go_build xvfb geckodriver dev_appserver_dep
 	# path before installing geckodriver as it includes version strings.
 	GECKODRIVER_PATH="$(shell find $(NODE_SELENIUM_PATH)geckodriver/ -type f -name '*geckodriver')"; \
 	cd webdriver; \
-	go test $(VERBOSE) -timeout=15m -tags=large -args \
+	COMMAND="go test $(VERBOSE) -timeout=15m -tags=large -args \
 		-firefox_path=$(FIREFOX_PATH) \
 		-geckodriver_path=$$GECKODRIVER_PATH \
 		-chrome_path=$(CHROME_PATH) \
 		-chromedriver_path=$(CHROMEDRIVER_PATH) \
 		-frame_buffer=$(USE_FRAME_BUFFER) \
 		-staging=$(STAGING) \
-		-test.timeout=30m \
-		-browser=$(BROWSER) $(FLAGS)
+		-browser=$(BROWSER) $(FLAGS)"; \
+	if [ "$$UID" == "0" ]; then sudo -u browser $$COMMAND; else $$COMMAND; fi
```

Comment:
Ok, so if we're not root we keep our current user; why not always switch to browser? For people running locally not in docker?

Reply:
On a second thought, if one attempts to run this outside of Docker, the command would fail because user `browser` does not exist. We don't officially support running the project without Docker but so far it has always been possible.

Reply:
Yeah, I suspected that was the case. Ok, lgtm with sadness :)